resources/core/charts/monitoring/charts/alertmanager/README.md

```diff
@@ -1,10 +1,10 @@
 ### Alertmanager
 
-In Kyma all the configuration related to the Alertmanager is in this chart.
+In Kyma part of the configuration related to the Alertmanager is in this chart.

```

Comment:
~In Kyma~ `This chart contains a part of the configuration related to the Alertmanager.`

Do we have to mention Kyma here?

Reply:
changed